Src/coffeescript/cql-execution/src/elm/quantity.coffee

```diff
@@ -29,6 +33,15 @@ clean_unit = (units) ->
 
 module.exports.createQuantity = (value,unit) ->
   new Quantity({value: value, unit: unit})
+  
+module.exports.parseQuantity = (str) ->
+  components = /(\d+\.?\d*)(.+)/.exec str
```

Comment:
Fixed in https://github.com/cqframework/clinical_quality_language/pull/35/commits/4732809275ed36d328d657bb7ce014cd7403d37e